creator-node/test/snapbackSM.test.js

```diff
@@ -1,23 +1,75 @@
 const nock = require('nock')
 const assert = require('assert')
 
-const { SnapbackSM, SyncType } = require('../src/snapbackSM/snapbackSM')
+const { SnapbackSM, SyncType, RECONFIG_MODES } = require('../src/snapbackSM/snapbackSM')
 const models = require('../src/models')
 const { getLibsMock } = require('./lib/libsMock')
 const utils = require('../src/utils')
 const { getApp } = require('./lib/app')
 const nodeConfig = require('../src/config')
 
 const constants = {
-  secondaryEndpoint: 'http://test_cn_2.co',
-  primaryEndpoint: 'http://test_cn.co',
-  primaryClockVal: 1
+  primaryEndpoint: 'http://test_cn_primary.co',
+  secondary1Endpoint: 'http://test_cn_secondary1.co',
+  secondary2Endpoint: 'http://test_cn_secondary2.co',
+  healthyNode1Endpoint: 'http://healthy1_cn.co',
+  healthyNode2Endpoint: 'http://healthy2_cn.co',
+  healthyNode3Endpoint: 'http://healthy3_cn.co',
+  primaryClockVal: 1,
+  wallet: '0x4749a62b82983fdcf19ce328ef2a7f7ec8915fe5',
+  userId: 1
 }
 
-describe('test sync queue', function () {
+const replicaSetNodesToUserClockStatusesMap = {}
+replicaSetNodesToUserClockStatusesMap[constants.primaryEndpoint] = {}
+replicaSetNodesToUserClockStatusesMap[constants.primaryEndpoint][constants.wallet] = 10
+replicaSetNodesToUserClockStatusesMap[constants.secondary1Endpoint] = {}
+replicaSetNodesToUserClockStatusesMap[constants.secondary1Endpoint][constants.wallet] = 10
+replicaSetNodesToUserClockStatusesMap[constants.secondary2Endpoint] = {}
+replicaSetNodesToUserClockStatusesMap[constants.secondary2Endpoint][constants.wallet] = 10
```

Comment:
nit why not define this map literal and all fields in a single assignment

Reply:
just what I thought of at the moment. no explicit reason